OpenRA.Mods.Common/Scripting/Properties/GeneralProperties.cs

```diff
@@ -73,11 +73,10 @@ public bool HasProperty(string name)
 			return Self.HasScriptProperty(name);
 		}
 
-		[Desc("Render a target flash on the actor. If set, 'asPlayer'",
-			"defines which player palette to use. Duration is in ticks.")]
-		public void Flash(int duration = 4, Player asPlayer = null)
+		[Desc("Render a target flash on the actor.")]
+		public void Flash(Color color, int count = 2, int interval = 2, int delay = 0)
 		{
-			Self.World.Add(new FlashTarget(Self, asPlayer?.Color ?? Color.White, duration));
+			Self.World.Add(new FlashTarget(Self, color, 0.5f, count, interval, delay));
```

Comment:
Can we get a comment here? Something like
```suggestion
			// TODO: We can't use floats with Lua, so use 0.5f as default here
			Self.World.Add(new FlashTarget(Self, color, 0.5f, count, interval, delay));
```